### PR TITLE
Doc: Style edit. Fix formatting, typos, usage.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/source/rest_api.rst [hunk 1]
```diff
@@ -49,7 +46,7 @@ Request Structure
 +------------+------------+-------------------------+
 | Field Name |    Type    |       Description       |
 +============+============+=========================+
-| name       | ``STRING`` |                         |
+| name       | ``STRING`` | Experiment name.        |
 |            |            | This field is required. |
 |            |            |                         |
 +------------+------------+-------------------------+
```
Comment (mateiz) [inline docs/source/rest_api.rst]: Were these auto-generated from a protobuf or written by hand? Would be better to change the protobuf if that's where they're from.

Comment (stbof) [reply docs/source/rest_api.rst]: The problem is that the generator has bugs. For the rest of the Databricks API docs we initially generate them from protobuf, but then we maintain them by hand.

Comment (mateiz) [reply docs/source/rest_api.rst]: I see. @mparkhe can you comment on this too? I'm okay switching to this but we need to make sure we don't regenerate them and lose the changes here.

Comment (mparkhe) [reply docs/source/rest_api.rst]: +1 to that. If there is a way to backfill the hand generated changes to PBs (after the said bugs are removed) that would be awesome. Esp., as we add more functionality and potentially more API changes to PBs, maintaining it by hand is painful.

**Comment by mparkhe** [comment]
LGTM

**Pull request merged**